src/org/usfirst/frc/team2706/robot/OI.java

```diff
@@ -48,19 +49,24 @@ public OI(Joystick driverStick, Joystick controlStick) {
         // Joystick for driving the robot around
         this.driverStick = driverStick;
 
+        // Joystick for controlling the mechanisms of the robot
+        this.controlStick = controlStick;
```

Comment:
This is supposed to be right above all of the controlStick commands, so at the bottom of this method